website/docs/usage/linguistic-features.mdx

```diff
@@ -194,7 +194,7 @@ information, without consulting the context of the token. The rule-based
 lemmatizer also accepts list-based exception files. For English, these are
 acquired from [WordNet](https://wordnet.princeton.edu/).
 
-### Trainable lemmatizer
+### Trainable lemmatizer {id="lemmatizer-train",model="lemmatizer"}
```

Comment:
I'm trying to think about what exactly the tags should do. Should this be `trainable_lemmatizer` then? (I might just punt and go back to `lemmatizer` on the section? I'm not sure how much this is confusing users.)

Reply:
Having it on the "lemmatizer" section is wrong, IMO. These tags define which trainable component should be in the pipeline, and the 3 subsections are quite different in that regard.

Reply:
So yes, let's make this `trainable_lemmatizer`.